packages/components/src/Badge/Badge.stories.tsx

```diff
@@ -34,3 +40,14 @@ export default meta;
 type Story = StoryObj<typeof meta>;
 
 export const Basic: Story = { render: args => <Badge {...args} /> };
+
+export const Icon: Story = {
+  parameters: {
```

Comment:
:-O that's so cool!

CC @sarahgm @OsamaAbdellateef

Reply:
good to know!